ResidentInformationApi/serverless.yml

```diff
@@ -22,6 +22,10 @@ functions:
     name: ${self:service}-${self:provider.stage}
     handler: ResidentInformationApi::ResidentInformationApi.LambdaEntryPoint::FunctionHandlerAsync
     role: lambdaExecutionRole
+    environments:
+      ACADEMYAPI_URL: ${ssm:/apiurls/academyapi-url}
+      HOUSINGAPI_URL: ${ssm:/apiurls/housingapi-url}
+      MOSAICAPI_URL: ${ssm:/apiurls/mosaicapi-url}
```

Comment:
Can we change these env variables to be `ACADEMY_API_URL`, `HOUSING_API_URL ` and `MOSAIC_API_URL` please